spec/graphql/static_validation/rules/required_arguments_are_present_spec.rb

```diff
@@ -10,7 +10,7 @@
     }
 
     fragment cheeseFields on Cheese {
-      similarCheese() { __typename }
+      similarCheese { __typename }
```

Comment:
I think static validation assumes that the query can be parsed, this test was failing because `context.visitor.visit` is failing. But if you took the whole query in this test and either parsed it or tried executing it, it raises the `ParseError` as expected.